spec/features/trainer_spec.rb

```diff
@@ -0,0 +1,63 @@
+require 'rails_helper'
+
+describe "training without cards" do
+  before :each do
+    create(:card)
+  end
+
+  it 'no cards' do
+    visit root_path
+    expect(page).to have_content 'Ожидайте наступления даты пересмотра.'
+  end
+end
+
+describe "training with some cards" do
+  before :each do
+    (0..1).each do
+      card = create(:card)
+      card.update_attribute(:review_date, Time.now)
+    end
+  end
+
+  it 'first visit' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.